pkg/views/util/text_wrapper.go

```diff
@@ -48,3 +48,11 @@ func GetTerminalWidth() int {
 	}
 	return width
 }
+func GetTerminalHeight() int {
```

Comment:
Please remove this util and do what you need inside of `calculateViewportSize`, we will figure out a way to get the term sizes properly, until then, it does not belong at this location

Reply:
done !